service-commands/scripts/init-repos.sh

```diff
@@ -32,18 +33,17 @@ npm install &
 # setup creator node
 cd $PROTOCOL_DIR/
 cd creator-node/
-npm install &
-npm install & #why does it not work without this?
+npm install
```

Comment:
was the solution here to just run npm install synchronously?

Reply:
yep - this seems to get rid of the issues w needing to install multiple times!